tools/build-docker.sh

```diff
@@ -23,8 +23,7 @@ fi
 
 # set the local image name if no name was provided
 localImageName=$2
-if [ -z "$localImageName" ]
-  then
+if [ -z "$localImageName" ]; then
```

Comment:
Could you (opportunistically) replace single brace expressions with double-brace?

https://google.github.io/styleguide/shellguide.html#test----and---

> [[ … ]] is preferred over [ … ], test and /usr/bin/[.
> 
> [[ … ]] reduces errors as no pathname expansion or word splitting takes place between [[ and ]]. In addition, [[ … ]] allows for regular expression matching, while [ … ] does not.

Reply:
done